staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1/types.go

```diff
@@ -90,6 +90,32 @@ type APIServiceSpec struct {
 
 	// leaving this here so everyone remembers why proto index 6 is skipped
 	// Priority int64 `json:"priority" protobuf:"varint,6,opt,name=priority"`
+
+	// ScopedResources declares a range of scoped resources to tell what resources to provide by the APIService. there can be more resources provided
```

Comment:
please remove the changes on the v1 version to make sure the existing models are not impacted after the authenticated review.